### PR TITLE
Fix particle rendering on the native platform is not correct.

### PR DESCRIPTION
**Pull request by caryliu1999**
Re: cocos-creator/2d-tasks#1381

### CODE REVIEW

File: builtin/jsb-adapter/HTMLCanvasElement.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ class HTMLCanvasElement extends HTMLElement {
         this._context2D = null;
         this._data = null;
         this._alignment = 4; // Canvas is used for rendering text only and we make sure the data format is RGBA.
+        //Whether the pixel data is premultiplied, the data returned from the native platform is all premultiplied.
```
Comment (jareguo) [inline builtin/jsb-adapter/HTMLCanvasElement.js]: 注意注释的空格，可参考文档 https://github.com/cocos-creator/creator-docs/blob/master/zh/scripting/reference/coding-standards.md

File: builtin/jsb-adapter/HTMLCanvasElement.js [hunk 2]
```diff
@@ -41,6 +41,8 @@ class HTMLCanvasElement extends HTMLElement {
...
+        this._premult = false;
```
Comment (jareguo) [inline builtin/jsb-adapter/HTMLCanvasElement.js]: 建议用过去式 _premultiplied

Comment (caryliu1999) [reply builtin/jsb-adapter/HTMLCanvasElement.js]: 已修改

File: builtin/jsb-adapter/HTMLCanvasElement.js (hunk 1 above)
Comment (caryliu1999) [reply builtin/jsb-adapter/HTMLCanvasElement.js]: 已修改

**Pull request merged**